lib/activity_notification/rails/routes.rb

```diff
@@ -153,7 +153,7 @@ def notify_to(*resources)
         if options[:with_devise].present? && options[:devise_default_routes].present?
           create_notification_routes options, resources_options
         else
-          self.resources target, only: :none do
+          self.resources target, only: [] do
```

Comment:
This is because of https://github.com/rails/rails/pull/51464 - all tests now pass, but I'm unsure of your intention @simukappu with this code

Reply:
hi @simukappu any feedback?

Reply:
I'm sorry for my very late response. I'm not sure with my original intent, but this modification seems fine. I'll merge your PR and update this gem. Thank you!

Reply:
@simukappu no problem, this gem is much appreciated.